### ISSUE TITLE
[Help]: Cannot find file landmask_ra.grd

### ISSUE THREAD

**Issue by arashfathollahi** [gmtsar/gmtsar#987]
Hello and good day,

I am encountering a "cannot find file landmask_ra.grd" error while processing the interferogram. Below are my system specifications:
GMTSAR 6.5
GMT 6.3
Ubuntu LTS 22.04.04
ASUS: CPU = AMD R7 6800H, RAM = 16 GB DDR5, GPU = RTX 3070
The issue arises specifically when I set the mask value to 1 in the batch_tops.config file to mask the wet regions. Interestingly, if I set the value to "no" so that masking does not occur, no error appears. All steps run correctly except for the landmask section.

It is worth mentioning that this error did not occur in the older versions of GMTSAR 6.0.0 and GMT 5.4.5 and Ubuntu LTS 20.04.04

I would appreciate any guidance you can provide. Thank you!
![Screenshot from 2024-08-30 20-41-33](https://github.com/user-attachments/assets/ff55138e-5854-4326-ba27-7aaf8fa3d5ee)
![Screenshot from 2024-08-30 23-47-01](https://github.com/user-attachments/assets/c9330714-810c-4b3c-ae1a-5e5df688e162)
![Screenshot from 2024-09-02 05-53-14](https://github.com/user-attachments/assets/3d0042e5-7f9d-420e-ac94-a2dc1d115bd8)

**Comment by Xiaohua-Eric-Xu** [comment]
I guess the real error is from "Killed" when landmask.csh is running. I am not sure what could kill the `gmt surface` command when it is running. Memory issue?

**Comment by arashfathollahi** [comment]
No, there is no problem with memory. Please guide me on how to solve the land mask issue. Even when executing the SBAS model command, entering the atom value of -3 results in a "killed" error.
![Screenshot from 2024-09-29 02-48-39](https://github.com/user-attachments/assets/c55b1221-c85f-41b6-93e5-3acc5fbe239d)

**Comment by arashfathollahi** [comment]
> I guess the real error is from "Killed" when landmask.csh is running. I am not sure what could kill the `gmt surface` command when it is running. Memory issue?

No, there is no problem with memory. Please guide me on how to solve the land mask issue. "Does it have no relation to the GMT & GMTSAR versions? What are the best versions currently for installation on Ubuntu 22.04?"

**Comment by Xiaohua-Eric-Xu** [comment]
Sorry for getting back to you late. I think the issue is most likely still memory. Some systems assigns limited memory for a single-core process. You could check that with `ulimit`. Or if you are running multiple processes that may take up all the memory. Or maybe it got interrupted somehow by other processes. I am not really sure.
